lib/flight_scheduler/commands/queue.rb

```diff
@@ -30,27 +30,54 @@ module Commands
     class Queue < Command
       extend OutputMode::TLDR::Index
 
-      register_column(header: 'JOBID') { |j| j.id }
-      register_column(header: 'PARTITION') { |j| j.partition.name }
-      register_column(header: 'NAME') { |j| j.attributes[:'script-name'] }
+      INCLUDES = ['partition', 'allocated-nodes', 'running-tasks', 'running-tasks.allocated-nodes']
+
+      register_column(header: 'JOBID') do |r|
+        if r.is_a?(JobsRecord) && r.attributes[:'last-index']
```

Comment:
The last job is being handled correctly:

![image](https://user-images.githubusercontent.com/287050/94439195-fe1dfe00-0197-11eb-95ab-4bc410f42635.png)

That should be `[3]` not `[3-3]`.

Reply:
Done in:
https://github.com/openflighthpc/flight-scheduler/pull/9/commits/a6f26e96337e45c7c6fff4a8af45adc0a838eabb